.circleci/config.yml

```diff
@@ -41,9 +41,6 @@ commands:
       - run:
           name: Black
           command: black sematic --diff --check
-      - run:
```

Comment:
`isort` and `black` were wanting to make conflicting changes to the code. As part of the overall modernization of dependencies, I would like to use `ruff` for import ordering, formatting and linting. This will replace `isort`, `flake8`, `black` with a single tool designed to handle all of these things without conflicts like this (and much faster as well). For now, we'll just ignore the isort check in CI, knowing that imports are well ordered now and will be again soon once we get the tooling itself sorted out.